tests/cover/test_numerics.py

```diff
@@ -23,11 +23,51 @@
 import pytest
 
 from hypothesis import given, assume
+from hypothesis.errors import InvalidArgument
 from tests.common.utils import fails
-from hypothesis.strategies import decimals, fractions
+from hypothesis.strategies import data, none, tuples, decimals, integers, \
+    fractions
 from hypothesis.internal.compat import float_to_decimal
 
 
+@given(data())
+def test_fuzz_fractions_bounds(data):
+    denom = data.draw(none() | integers(1, 20))
+    fracs = none() | fractions(max_denominator=denom) \
+        | fractions('1/99', '1/2', denom)
```

Comment:
It makes me sad that there's no equivalent to `@example` for `data`. I've still yet to figure out a good API for it.